src/components/content/content.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { ComponentStory } from "@storybook/react";
+
+import Content from ".";
+import Typography from "../typography";
+
+export const DefaultStory: ComponentStory<typeof Content> = (args) => (
+  <Content title="Title" {...args}>
+    This is an example of some content
+  </Content>
+);
+
+export const InlineContent = DefaultStory.bind({});
+InlineContent.args = { inline: true };
+
+export const CustomTitle = DefaultStory.bind({});
```

Comment:
Thanks for updating these @DipperTheDan! 

I did notice when running storybook the rendered source code for this story shows `Typography` as `styled.span`. You can fix this by explicitly adding the `displayName` in `Typography`'s component file

![Screenshot 2022-09-28 at 12 00 27](https://user-images.githubusercontent.com/18368713/192762724-93fd5f6f-ac3e-424a-8e9d-12ff3b1213cd.png)

Reply:
`styled.span` is exactly what I wanted it to say 👀 
🤣 Fixed it.